VMBackup/main/guestsnapshotter.py

```diff
@@ -94,6 +94,8 @@ def snapshot(self, sasuri, sasuri_index, meta_data, snapshot_result_error, snaps
                         key = meta['Key']
                         value = meta['Value']
                         headers["x-ms-meta-" + key] = value
+                if(CommonVariables.isSnapshotTTLEnabled in settings):
+                    headers["x-ms-snapshot-ttl-expiry-hours"] = settings[CommonVariables.isSnapshotTTLEnabled]
```

Comment:
settings[CommonVariables.isSnapshotTTLEnabled] will return true/false. I believe the value to set for x-ms-snapshot-ttl-expiry-hours is 1.